keyboards/bioi/c11/rules.mk

```diff
@@ -0,0 +1,34 @@
+MCU = atmega32u4
+F_CPU = 8000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+BOOTLOADER = qmk-dfu
+BOOTLOADER_SIZE = 4096
+
+
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
```

Comment:
This can be removed.   Just the processor and bootloader need to be specified, and in the info.json file.

Reply:
Some of the design is using ATmega32u4 under 8Mhz@3.3V, I can't find any document that instructing me how to define that in a info.json, and some design using a custom built qmk-dfu with a size of 8192, I think I should keep some definition here.